benchmarks/src/has_single_bit.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <assert.h>
+#include <benchmark/benchmark.h>
+#include <bit>
+
+#include <utility.hpp>
+
+template <typename T>
+static void has_single_bit_if(benchmark::State& state) {
+    const auto random_v = random_vector<T>(8);
+    for (auto _ : state) {
+        benchmark::DoNotOptimize(random_v);
+        int count_true = 0, count_false = 0;

```

Comment:
We conventionally avoid multiple variables in a single declaration, even for the non-problematic cases.

Reply:
done.